mobile/test/cc/unit/fetch_client_test.cc

```diff
@@ -18,10 +18,14 @@ TEST(FetchClientTest, Http2) {
 TEST(FetchClientTest, Http3) {
   Envoy::Fetch client;
   std::vector<Http::Protocol> protocols;
-  ASSERT_EQ(client.fetch({"https://www.google.com/"}, {"www.google.com"}, protocols),
+  ASSERT_EQ(client.fetch({"https://www.google.com/", "https://www.google.com/"}, {"www.google.com"},
+                         protocols),
             ENVOY_SUCCESS);
-  // TODO(fredyw): In CI, HTTP/3 does not work and will use HTTP/2 instead.
-  ASSERT_GT(protocols.front(), Http::Protocol::Http11);
+  // The first request could either be HTTP/2 or HTTP/3 because we no longer give HTTP/3 a head
+  // start.
+  ASSERT_GE(protocols.at(0), Http::Protocol::Http2);
+  // TODO(fredyw): In EngFlow CI, HTTP/3 does not work and will use HTTP/2 instead.
```

Comment:
Wow, that's interesting. Any idea why yet?

Reply:
I don't know yet. I pinged EngFlow in Slack. I suspect UDP is blocked or something even after setting https://docs.engflow.com/re/client/platform-options-reference.html#dockernetwork